src/main/java/taxi/service/DriverService.java

```diff
@@ -1,6 +1,11 @@
 package taxi.service;
 
+import java.util.Optional;
 import taxi.model.Driver;
 
 public interface DriverService extends GenericService<Driver> {
+    Optional<Driver> findByLogin(String login);
+
+    Driver register(String login, String password,
+                    String repeatPassword, String name, String licenseNumber);
```

Comment:
it's better to use DriverService create method and make check of passwords in controller